alpine/Dockerfile

```diff
@@ -20,8 +20,6 @@ RUN set -eux; \
 	esac; \
     if [ "$ASSET" = "ce" ] ; then \
         apk add --no-cache --virtual .build-deps curl wget tar ca-certificates && \
-        curl -fL "https://bintray.com/kong/kong-alpine-tar/download_file?file_path=kong-$KONG_VERSION.$arch.apk.tar.gz" -o /tmp/kong.tar.gz && \
-        echo "$KONG_SHA256  /tmp/kong.tar.gz" | sha256sum -c -; \
         apk del .build-deps; \
```

Comment:
A final implementation would add the s390x arch as line 20 and not delete current lines 23 and 24.

Reply:
Gotcha. 
So these PRs will let us build "locally" so to speak and we can then update to align with the standard process once we've got an artifact in bintray?